kubernetes/config/default/manager_webhook_patch.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
```

Comment:
I'm guessing these are all auto-generated? Because not sure we need things like webhooks for now.

Reply:
config/ folder are auto generated and we can clean up some of them not in used now. 

Using webhook as an example, it is recommended to patch job with some default value or doing some validation. At this moment, we have not used it yet. 

1. We can delete these manifest and add them back whenever we add the corresponding suport 
2. We can keep them there. 

Which way do you prefer?

Reply:
I'd prefer to keep the PR scoped down and add features down the line. Is this hard to do?

Reply:
Yeah. I will do that